### ISSUE TITLE
Attempting to build on Centos 7 - get " src/network/v3/language-agent/Tracing.pb.h: No such file or directory"

### ISSUE THREAD

**Issue by mfurlend** [SkyAPM/SkyAPM-php-sdk#289]
**Describe the bug**
When I run `make` on Centos 7 I get this error:
```
[root@52794644a6c8 skywalking]# make
/bin/sh /tmp/skywalking/libtool --mode=compile g++ -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/tmp/skywalking -DPHP_ATOM_INC -I/tmp/skywalking/include -I/tmp/skywalking/main -I/tmp/skywalking -I/opt/remi/php73/root/usr/include/php -I/opt/remi/php73/root/usr/include/php/main -I/opt/remi/php73/root/usr/include/php/TSRM -I/opt/remi/php73/root/usr/include/php/Zend -I/opt/remi/php73/root/usr/include/php/ext -I/opt/remi/php73/root/usr/include/php/ext/date/lib -I/usr/local/include -I/tmp/skywalking/src -I/tmp/skywalking/src/network/v3  -DHAVE_CONFIG_H  -g -O2 -Wall -std=c++11 -Wno-deprecated-register   -c /tmp/skywalking/skywalking.cc -o skywalking.lo
libtool: compile:  g++ -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/tmp/skywalking -DPHP_ATOM_INC -I/tmp/skywalking/include -I/tmp/skywalking/main -I/tmp/skywalking -I/opt/remi/php73/root/usr/include/php -I/opt/remi/php73/root/usr/include/php/main -I/opt/remi/php73/root/usr/include/php/TSRM -I/opt/remi/php73/root/usr/include/php/Zend -I/opt/remi/php73/root/usr/include/php/ext -I/opt/remi/php73/root/usr/include/php/ext/date/lib -I/usr/local/include -I/tmp/skywalking/src -I/tmp/skywalking/src/network/v3 -DHAVE_CONFIG_H -g -O2 -Wall -std=c++11 -Wno-deprecated-register -c /tmp/skywalking/skywalking.cc  -fPIC -DPIC -o .libs/skywalking.o
In file included from /tmp/skywalking/skywalking.cc:23:0:
/tmp/skywalking/src/segment.h:23:54: fatal error: src/network/v3/language-agent/Tracing.pb.h: No such file or directory
 #include <src/network/v3/language-agent/Tracing.pb.h>
```
And there is indeed no such file, and I can't seem to find it anywhere.

**To Reproduce**
Attempt to build on Centos 7

**Expected behavior**
PHP SDK builds

**Comment by heyanlong** [comment]
https://github.com/SkyAPM/SkyAPM-php-sdk/blob/master/docs/install.md

**Comment by NobleNoob** [comment]
同样的问题发生在 centos 7 
```
libtool: compile:  g++ -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/SkyAPM-php-sdk-4.0.0 -DPHP_ATOM_INC -I/SkyAPM-php-sdk-4.0.0/include -I/SkyAPM-php-sdk-4.0.0/main -I/SkyAPM-php-sdk-4.0.0 -I/usr/local/php7/include/php -I/usr/local/php7/include/php/main -I/usr/local/php7/include/php/TSRM -I/usr/local/php7/include/php/Zend -I/usr/local/php7/include/php/ext -I/usr/local/php7/include/php/ext/date/lib -I/usr/local/include -I/SkyAPM-php-sdk-4.0.0/src -I/SkyAPM-php-sdk-4.0.0/src/network/v3 -DHAVE_CONFIG_H -g -O2 -Wall -std=c++11 -Wno-deprecated-register -c /SkyAPM-php-sdk-4.0.0/skywalking.cc  -fPIC -DPIC -o .libs/skywalking.o
In file included from /SkyAPM-php-sdk-4.0.0/skywalking.cc:22:0:
/SkyAPM-php-sdk-4.0.0/src/segment.h:23:54: fatal error: src/network/v3/language-agent/Tracing.pb.h: No such file or directory
 #include <src/network/v3/language-agent/Tracing.pb.h>
                                                      ^
compilation terminated.
make: *** [skywalking.lo] Error 1
```

**Comment by NobleNoob** [comment]
The issue has been resolved.
I think Centos 7 would not define the shared library path by default which is 'grpc_cpp_plugin' need it.
From my side just adding an environment variable and its work.
` export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:/usr/local/lib64/`
